com.unity.probuilder/Editor/EditorCore/RepairActions.cs

```diff
@@ -139,17 +136,17 @@ public static void MenuRemoveDegenerateTriangles()
 
             foreach (ProBuilderMesh pb in InternalUtility.GetComponents<ProBuilderMesh>(Selection.transforms))
             {
-                pb.ToMesh();
-
-                int[] rm = pb.RemoveDegenerateTriangles();
-                count += rm != null ? rm.Length : 0;
+                var removed = MeshValidation.EnsureMeshIsValid(pb);
```

Comment:
returned value is not documented properly, change method signature or document correctly

Reply:
14789d9c